### PR TITLE
Fill Processing Utils

### PR DESCRIPTION
**Pull request by maerch**
Extracts generic fill processing logic into the logic package. The background removal-specific code is reduced to the actual processing only. After this is merged, very little code has to be copied for the vectorizer or any other fill processing plugin.

### CODE REVIEW

File: packages/plugin-utils/src/processing/processFill.ts [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+import { type Source } from '@cesdk/cesdk-js';
+import type FillProcessingMetadata from '../metadata/FillProcessingMetadata';
+import { Optional } from '../types/Optional';
+
+export default async function processFill<T>(
+  cesdk: CreativeEditorSDK,
+  blockId: number,
+  metadata: FillProcessingMetadata,
+  process: (
+    sources: Optional<Source, 'width' | 'height'>[],
```
Comment (mirko314) [inline packages/plugin-utils/src/processing/processFill.ts:11]: I'm not quite sure what `sources` exactly mean

**Review by mirko314 — APPROVED** [review]
LGTM, only skimmed over though

File: packages/plugin-utils/src/processing/processFill.ts (hunk 1 above)
Comment (maerch) [reply packages/plugin-utils/src/processing/processFill.ts:11]: https://img.ly/docs/cesdk/engine/guides/source-sets/

Comment (mirko314) [reply packages/plugin-utils/src/processing/processFill.ts:11]: I see, so `sources` is a  `sourceSet`? Would it maybe make more sense to call it `SourceSet` so it's discoverable?

Comment (maerch) [reply packages/plugin-utils/src/processing/processFill.ts]: Done

**Pull request merged**